TwitchLib.Client/Throttling/ThrottlingService.cs

```diff
@@ -65,13 +65,13 @@ internal bool Enqueue(OutboundChatMessage message)
             _queue.Enqueue((DateTime.UtcNow, message));
             return true;
         }
-        
-        private Task StartThrottlerAsync(object sender, OnConnectedEventArgs args)
+
+        private Task StartThrottlerAsync(object sender, Communication.Events.OnConnectedEventArgs args)
         {
             // Cancel old token first
             _tokenSource.Cancel();
             _tokenSource = new CancellationTokenSource();
-            _sendTask= Task.Run(async() => await SendTaskActionAsync(), _tokenSource.Token);
+            _sendTask = Task.Run(async () => await SendTaskActionAsync(), _tokenSource.Token);
```

Comment:
Perhaps this should have been the following?
```suggestion
            _sendTask = SendTaskActionAsync(_tokenSource.Token);
```
Keep in mind that the token passed to Task.Run will only cancel the "wait handle" and not the delegate execution asynchronous or otherwise unless you explicitly flow the token down the call stack.